### PR TITLE
Add schema updates for sovereign cloud support

### PR DESCRIPTION
**Pull request by danieljurek**
Tracks https://github.com/Azure/azure-dev/pull/3452

### CODE REVIEW

File: schemas/v1.0/azure.yaml.json [hunk 1]
```diff
@@ -525,6 +525,15 @@
                     "$ref": "#/definitions/workflow"
                 }
             }
+        },
+        "cloud": {
+            "type": "object",
+            "title": "The cloud configuration used for the project.",
+            "description": "Optional. Provides additional configuration for deploying to sovereign clouds such as Azure Government.",
```
Comment (rajeshkamal5050) [inline schemas/v1.0/azure.yaml.json:532]: Default is AzureCloud? Shouldn't we also say that?

**Review by wbreza — APPROVED** [review]
Looks good - Lets just hold on merging till right before the release.

File: schemas/v1.0/azure.yaml.json (hunk 1 above)
Comment (danieljurek) [reply schemas/v1.0/azure.yaml.json]: Description updated.

**Pull request merged**